spec/features/paginate_posts_index_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe 'Paginate posts index' do
+  subject { page }
+  describe 'index is paginated' do
+    before do
+      25.times { FactoryGirl.create(:blog_post) }
```

Comment:
Done. These distinctions are important, and I'd like to learn more about defining constants.